app/components/code-snippet.js

```diff
@@ -27,15 +30,24 @@ export default Ember.Component.extend({
   },
 
   source: Ember.computed('name', function(){
-    return this._unindent(
+    const source = this._unindent(
       (Snippets[this.get('name')] || "")
         .replace(/^(\s*\n)*/, '')
         .replace(/\s*$/, '')
     );
+    if (this.get('lineNumbers')) {
+      const lang = this.get('language');
+      return codeHighlightLinenums(source, {hljs:Highlight, lang, start:1 })
+    }
+
+    return source;
+
   }),
 
   didInsertElement: function(){
-    Highlight.highlightBlock(this.get('element'));
+    if(!this.get('lineNumbers')) {
+      Highlight.highlightBlock(this.get('element'));
+    }
```

Comment:
For consistency, let's eliminate didInsertElement and always render `{{{source}}}`, regardless of whether lineNumbers is true or false. 

Otherwise the fastboot behavior is different based on whether you set lineNumbers, which would be surprising.